modules/_labs/header/react/lib/Header.tsx

```diff
@@ -191,45 +175,30 @@ const navStyle = ({themeColor}: Pick<HeaderProps, 'themeColor'>) => {
   });
 };
 
-const ChildrenSlot = styled('div')<Pick<HeaderProps, 'breakpoints' | 'centeredNav' | 'themeColor'>>(
-  ({centeredNav = false, breakpoints}) => {
-    const mq = makeMq(breakpoints);
-
-    return {
-      marginRight: spacing.m,
-
-      // TODO: remove this when we get real icon buttons
-      '> .canvas-header--menu-icon': {
-        cursor: 'pointer',
-      },
-      [mq.sm]: {
-        display: 'flex',
-        alignItems: 'center',
-        justifyContent: 'flex-end',
-        height: '100%',
-        marginRight: spacing.m,
-
-        '> *': {
-          marginLeft: childrenSpacing,
-        },
-        '> *:not(.canvas-header--menu-icon)': {
-          display: 'none',
-        },
-      },
-      [mq.md]: {
-        '> *:last-child': {
-          marginRight: 0,
-        },
-
-        '> *:not(.canvas-header--menu-icon)': {
-          display: 'flex',
-        },
-      },
-      [mq.lg]: {
-        flexGrow: centeredNav ? 1 : 'unset',
-      },
-    };
+const ChildrenSlot = styled('div')<Pick<HeaderProps, 'centeredNav' | 'themeColor' | 'isCollapsed'>>(
+  {
+    marginRight: spacing.m,
+    // TODO: remove this when we get real icon buttons
```

Comment:
Do we have real icon buttons yet?

Reply:
TBH I have no idea, but it seemed out of scope for this PR